File: CapitalesAfricaJava/app/src/main/java/com/example/capitalesafricajava/MainActivity.java
package com.example.capitalesafricajava;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.RelativeLayout;

public class MainActivity extends AppCompatActivity {

    Intent juego;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button comenzar = (Button) findViewById(R.id.botonComenzar);
        DbHandler dbHandler = new DbHandler(MainActivity.this);
        dbHandler.deleteAll();


        comenzar.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {

                juego = new Intent(MainActivity.this, Juego.class);
                startActivity(juego);
            }
        });
    }
}

File: CapitalesAfricaKotlin/app/src/main/java/com/example/capitalesafricakotlin/MainActivity.kt
package com.example.capitalesafricajava

import androidx.appcompat.app.AppCompatActivity
import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import com.example.capitalesafricajava.R

class MainActivity : AppCompatActivity() {
    var juego: Intent? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val comenzar = findViewById<View>(R.id.botonComenzar) as Button
        val dbHandler = com.example.capitalesafricajava.DbHandler(this@MainActivity)
        dbHandler.deleteAll()
        comenzar.setOnClickListener {
            juego = Intent(this@MainActivity, Juego::class.java)
            startActivity(juego)
        }
    }
}